tests/services/inventory/test_data/mock_gcp_results.py

```diff
@@ -121,6 +121,52 @@
     ]
 }
 
+AD_GET_GROUPS_SETTINGS = {
```

Comment:
You need to have a copy of this template for each group returned by AD_GET_GROUPS above.